src/de/jost_net/JVerein/gui/action/MitgliedskontoExportAction.java

```diff
@@ -89,7 +89,7 @@ public void handleAction(Object context) throws ApplicationException
       Logger.error("Fehler", e);
       GUI.getStatusBar().setErrorText(
 
-      "Fehler beim exportieren der Mitgliedskonten");
+      "Fehler beim exportieren der Rechnungen");
```

Comment:
Das müsste dann auch wieder rückgängig gemacht werden.

Reply:
Nein, es sollte Sollbuchungen heißen und nicht Mitgliedskonten.
Evtl. sollte man die Klasse Mitgliedskonto in Sollbuchung abändern, aber es sind sehr viele Stellen die man anpassen müsste. Aber ich kann das ja mal später probieren.